extensions/kubernetes-client/runtime/pom.xml

```diff
@@ -30,6 +30,10 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-httpclient-okhttp</artifactId>
```

Comment:
Shouldn't we use the JDK implementation anyway?

Reply:
The JDK version is not perfect and there was an issue with JDK<16 and WebSockets (IIRC).
Also this will probably open another can of words. Similar to the TODO comment, I would try to tackle the replacement somewhere else.